HISTORY.rst

```diff
@@ -24,6 +24,10 @@ Internal changes
 * Typing syntax has been updated within pre-commit via `isort`. Pre-commit hooks now append `from __future__ import annotations` to all python module imports for backwards compatibility. (:issue:`1065`, :pull:`1071`)
 * `isort` project configurations are now set in `setup.cfg`. (:pull:`1071`).
 
+Bug fixes
+^^^^^^^^^
+* When an argument of `xclim.sdba.adjust` is a `xr.DataArray`, only print the name instead of the whole array. (:issue:`1083`, :pull:`1087`)
```

Comment:
```suggestion
* When an argument of `xclim.sdba.adjust` is an `xr.DataArray`, only print the name instead of the whole array. (:issue:`1083`, :pull:`1087`).
```